<?xml version="1.0"?>
<launch>

    <!-- Load parameters -->
    <rosparam command="load" ns="" file="$(find grasping_pipeline)/config/standard.yaml"/>
    <rosparam command="load" ns="" file="$(find placement)/config/startup.yaml"/>
    <rosparam command="load" ns="" file="$(find table_plane_extractor)/config/config.yaml"/>
    <node name="table_plane_extractor_server" pkg="table_plane_extractor" type="table_plane_extractor_srv.py"/>
    <node name="CollisionEnvironment" pkg="grasping_pipeline" type="collision_environment.py"/>
    <node name="execute_grasp_server" pkg="grasping_pipeline" type="execute_grasp_action_server.py"/>
    <node name="find_grasppoint_server" pkg="grasping_pipeline" type="find_grasppoint_action_server.py"/>
    <node name="handover_server" pkg="handover" type="handover_server.py"/>
    <node name="grasping_statemachine" pkg="grasping_pipeline" type="statemachine.py"  output="screen"/>
    <!--<node name="map_cloud_publisher" pkg="grasping_pipeline" type="map_cloud_publisher.py" output="screen"/>-->
</launch>
